spec/game_spec.rb

```diff
@@ -102,4 +102,73 @@
         })
     end
   end
+
+  describe '#check_win' do
+    before(:each) do
+      @new_game = Game.new
+    end
+
+    it 'returns win for a horizontal row of 4' do
+      @new_game.new_board.grid = {
+        row_0: "ABCDEFG",
+        row_1: ".......",
+        row_2: ".......",
+        row_3: ".......",
+        row_4: ".......",
+        row_5: ".......",
+        row_6: ".XXX...",
+        }
+
+        @new_game.take_turn(3)
+
+        expect(game.check_win).to be True
+    end
+
+    it 'returns win for a vertical row of 4' do
+      @new_game.new_board.grid = {
```

Comment:
Nice. We should refactor these to have the 'O's like the next tests, other than that, looks good. Now comes the hard part!